src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -122,6 +154,23 @@ protected function getReceiver(): ReceiverInterface
 
     protected function getLastRedeliveryStampWithException(Envelope $envelope): ?RedeliveryStamp
     {
+        if (null === \func_get_args()[1]) {
+            trigger_deprecation(
+                'symfony/messenger',
+                '5.1',
```

Comment:
Should be 5.2 - there are a bunch of these spots that need to go from 5.1 to 5.2 in the PR

Reply:
Ah, yes. You're right. I'll search and update them.